modules/cluster-logging-loki-deploy.adoc

```diff
@@ -0,0 +1,133 @@
+// Module is included in the following assemblies:
```

Comment:
@libander I've downloaded this PR and built it locally, and I am not seeing this file's content included in my local build. Maybe I'm missing something somewhere. Any ideas?